cypresse2e/cypress/elements/pages/aboutUs.js

```diff
@@ -13,6 +13,8 @@ class AboutUs {
   joinStudentProgramLink = "https://apply.workable.com/fightpandemics/j/58B157AAB2/";
   getToHelpBoardButton = "#AU_VHB";
   goToHelpBoardLink = "/feed";
+  getToPressKitButton = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[9]/div/a";
```

Comment:
Could you try this CSS selector: div[direction="column"] > a[role="button"] or div[direction="column"] > a[role="button"] span